docs/source/conf.py

```diff
@@ -942,6 +942,7 @@
     "guard_scalar",
     "has_hint",
     "has_symbolic_sizes_strides",
+    "guard_size_oblivious",
```

Comment:
Please no new entry should be added here.
Either make it private or make sure it is properly a exposed as a public API (in this case, adding it to https://github.com/pytorch/pytorch/blob/08a7af37f1b4a04695b8cc2ba061c29c110e41d1/torch/fx/experimental/sym_node.py#L45). Also make sure it is not accidentally as a public API in another module by adding the appropriate `__all__` there or removing it from the module public space.

Reply:
You're looking at the wrong file; this is already in `__all__`. You got your wish; I added a doc page for symbolic shapes lol.